docs/COC.md

```diff
@@ -0,0 +1,55 @@
+
+# Ethics Statement
+
+All members of the OBO community will be expected to abide by the highest ethical standards, for example, in accordance with the NIH standards for ethical conduct (https://oir.nih.gov/sourcebook/ethical-conduct). Plagiarism will not be tolerated. 
+
+# Diversity Statement
+We encourage every member of the OBO community to participate, and we are committed to building a community for all. Although we may not always be successful, we seek to treat every person as fairly and equally as possible. If a community member makes a mistake, then we expect that person to own responsibility for the error and correct it. If a person has been harmed or offended as a result of another person’s action, and if a community member is witness, then both members own responsibility to listen carefully and respectfully to person who has been harmed and make every effort to not repeat the behaviour, as well as to make every effort to correct the damage.
+
+Although this list cannot be exhaustive, we explicitly honor diversity in age, gender, gender identity or expression, culture, ethnicity, language, national origin, political beliefs, profession, race, religion, sexual orientation, socioeconomic status, and technical ability. We will not tolerate discrimination based on any of the protected characteristics above, including participants with disabilities.
+
+# Conflict Resolution and Reporting Issues
+
+**Conflict resolution team**
+
+The OBO-COC team is intentionally small with only three members, in order to best support confidential handling of issues. Members of the OBO-COC team are nominated by the community and voted on by the OBO-Operations committee. The term shall be 2 years, but may be renewed.
+
+**Reporting issues** 
+
+If a community member experiences or witnesses unacceptable behavior, or if any other concern arises, then please report it by contacting a member of the OBO-COC team. If the issue involves a member of the COC-team, then please contact a different member. If these options are not sufficient, consider contacting a member of the OBO-Operations committee and/or your local ombuds office.
+All reports to OBO-COC will be handled with discretion and will receive serious attention. Reports should include the following:
+
+Name of person submitting the report.
+Names (real, nicknames, or pseudonyms) of any individuals involved. If there are additional witnesses, please include them as well. An account of the incident and whether it is ongoing. A link to any publicly available record (e.g., a mailing list archive, ticket, or a public Slack log), if applicable.
```

Comment:
This might be easier to read as a bullet list

Reply:
+1

Reply:
the comment about plagiarism doesn't seem relevent and might even be confusing given that we encourage reuse.